README.md

```diff
@@ -78,6 +78,55 @@ Contributions to this project are encouraged and welcomed! Your expertise in cod
 A successful build will display "Project built :)".
 
 
+## Docker Container Development
+
+This needs to have Wine32 Installed on your Linux machine prior to running this (at least in the current version of this README file)
```

Comment:
Wine32 is not need when using the docker container :)

```suggestion
```

Reply:
the docker container already has wine32 there?

Reply:
Yup, see https://github.com/lhsazevedo/tokyo-bus-guide-decomp/blob/main/docker/Dockerfile#L22C1-L25C17

Reply:
done!